### DISCUSSION TITLE
Is Github Copilot available for Visual Studio 2019, 2017?

### DISCUSSION THREAD

**Discussion by yensubldg** [community/community#33073] [Copilot Conversations] [answered]
I try install Github Copilot on VS 2019 but I can find this on market extension in VS 2019

**Comment by samuel-ma** [comment]
@yensubldg 
Github Copilot is only available for VS Code unfortunately...
I love using Visual Studio 2019 for my C++ projects, also wish copilot was also compatible with it

**Comment by davecheney** [comment, marked as answer]
No.
